Makefile

```diff
@@ -1,5 +1,7 @@
+# Unconditionally make all targets; they're all phony
+MAKEFLAGS += --always-make
```

Comment:
Hmm, while I appreciate your reasons for it, I'd rather add `PHONY` where it's missing. These things don't even change that often and are not too disruptive if we do find an issue every now and then.

That said, it's ok with me if you really prefer it this way.

Reply:
Yeah, it's just that we don't really use make to *build* anything. It's only for running a bunch of recipes, really. So everything is "phony". If I could get away with switching to something like https://github.com/casey/just, which is basically make but without any of the build stuff and all the historical baggage, I'd consider it.

Reply:
The alternative to adding `.PHONY: foo` for each target `foo` is to make each target depend on a `FORCE` target. See https://www.gnu.org/software/make/manual/make.html#Force-Targets

So instead of

```
.PHONY: clean
clean:
        ...
```
We'd have

```
clean: FORCE
        ...
```

That's a bit easier on the eye, perhaps?

Reply:
Well, we could end up creating rules to build packages or to do code generation... Things change. And then we'll need to add these back after someone notices that things are being regenerated when they shouldn't, which is much more puzzling to debug by the way.

`always-make` would be the last assumption I would make as to why something was being regenerated when I didn't want it to. Especially as it's so far removed from the rule definition itself. In fact, I'd only use that switch explicitly and for debugging purposes. Maybe. Conversely, something doesn't run, I know where to look.

Anyway, it really is up to you.

Reply:
Fair enough. I assume you wrote that before my suggestion about `FORCE` - compromise?

Reply:
> The alternative to adding `.PHONY: foo` for each target `foo` is to make each target depend on a `FORCE` target. See https://www.gnu.org/software/make/manual/make.html#Force-Targets

From the manual: `Using ‘.PHONY’ is more explicit and more efficient.` :sweat_smile: 

Seriously, if you don't like make, we should be considering meson or something like that. But as long as we're in Rome...

Reply:
> Fair enough. I assume you wrote that before my suggestion about `FORCE` - compromise?

I did. And the following one before I saw this one too. I don't think it's a good compromise by the way. This is the worst option so far: doesn't solve your issue and is equally bad from my perspective.

Reply:
Okay, okay! :smile: 

Dropped this, updated .PHONY deps, and split out the src_dir change to a separate commit.